src/Waypoint/Directive.js

```diff
@@ -10,11 +23,19 @@ var zumWaypoint = function zumWaypoint($window, WaypointService) {
 		link : function zumWaypointLink(scope, element, attrs, ctrl) {
 			var callback = angular.bind(ctrl, ctrl.processWaypoint);
 			/*jshint -W031 */
```

Comment:
@earlyster This jshint directive informs the linter to stop complaining about the constructor side effects below.  It should remain as close to the closing directive as possible.  Since you've moved the options object out of the instantiation statement, can you also move this directive down to line 37-ish?